app/models/mediaflux/request.rb

```diff
@@ -14,8 +14,12 @@ def self.request_path
       end
 
       def self.uri
-        # This is the necessary URI for the Ansible build(s) of Mediaflux
-          URI("#{Connection.transport}://#{Connection.host}/#{request_path}")
+        # Setting the URI to the Ansible build or Docker build of mediaflux depending on the environment
+        if ENV["MFLUX_LOCAL"]
+          URI("#{Connection.transport}://#{Connection.host}:#{Connection.port}#{request_path}")
```

Comment:
I wonder if the port could just be added.  It seems weird that we are not allowing mediaflux to be on a different port...

Reply:
True — we can just change the port too when we want to run against docker. I can push an update shortly